src/controllers/hello-world.api.ts

```diff
@@ -1,30 +1,24 @@
-export const def = {
-    basePath: '/',
-    paths: {
-        '/helloworld': {
-            get: {
-                'x-operation-name': 'helloWorld',
-                parameters: [
-                    {
-                        name: 'name',
-                        in: 'query',
-                        description: 'Your name.',
-                        required: false,
-                        type: 'string',
-                    },
-                ],
-                responses: {
-                    200: {
-                        schema: {
-                            response: {
-                                type: 'string',
-                                description: 'Returns a hello world with your (optional) name.',
-                            },
-                        },
-                    },
-                },
-            },
-        },
-    },
-};
-
+export const apiSpec =
+{
+  "paths": {
+    "/helloworld": {
+      "get": {
+        "x-operation-name": "helloWorld",
+        "parameters": [
+          {
+            "name": "name",
+            "in": "query",
+            "description": "Your name.",
+            "required": false,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Returns a hello world with your (optional) name."
```

Comment:
My suggestion was to buff up the API spec with what we return. I realize though that only the `description` is required for the a [Response Object](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#response-object), so you can disregard my comment if you want to keep it simple.